libraries/botbuilder-azure/package.json

```diff
@@ -47,9 +47,10 @@
     "build": "tsc -b",
     "build-docs": "typedoc --theme markdown --entryPoint botbuilder-azure --excludePrivate --includeDeclarations --ignoreCompilerErrors --module amd --out ..\\..\\doc\\botbuilder-azure .\\lib\\index.d.ts --hideGenerator --name \"Bot Builder SDK - Azure\" --readme none",
     "build:rollup": "yarn clean && yarn build && api-extractor run --verbose --local",
-    "clean": "rimraf _ts3.4 lib tsconfig.tsbuildinfo",
+    "clean": "rimraf _ts3.4 lib vendors tsconfig.tsbuildinfo",
     "depcheck": "depcheck --config ../../.depcheckrc",
     "lint": "eslint .",
+    "prebuild": "tsup ./node_modules/p-map/*.js --format cjs --dts --out-dir vendors/p-map --clean --sourcemap",
```

Comment:
Add the vendors folder to the files property in this package.json like...
![imagen](https://github.com/user-attachments/assets/2aed328f-aef1-481b-8a3b-2a914b04d703)

Reply:
fixed